src/request/derive-address/index.html

```diff
@@ -109,14 +109,14 @@ <h1 data-i18n="derive-address-heading-password" class="nq-h1">Unlock your accoun
                     <div class="identicon"></div>
                     <div class="address"></div>
                 </div>
-                <button class="nq-button light-blue confirm" data-i18n="derive-address-add-to-account-button">Add to account</button>
+                <button class="nq-button light-blue confirm" data-i18n="derive-address-add-to-account-button">Add to Account</button>
             </div>
             <div id="effect-container">
                 <div class="page-header nq-card-header">
                     <a tabindex="0" class="page-header-back-button display-none">
                         <svg class="nq-icon"><use xlink:href="../../../node_modules/@nimiq/style/nimiq-style.icons.svg#nq-arrow-left"/></svg>
                     </a>
-                    <h1 data-i18n="derive-address-heading-choose-identicon" class="nq-h1">Choose a new address</h1>
+                    <h1 data-i18n="derive-address-heading-choose-identicon" class="nq-h1">Choose a new Avatar</h1>
```

Comment:
@julianbauer 
If we remove `a` and `an` everywhere, why not here?
```suggestion
                    <h1 data-i18n="derive-address-heading-choose-identicon" class="nq-h1">Choose new Avatar</h1>
```